soc/arm/nxp_lpc/lpc54xxx/gcc/startup_LPC54114_cm4.S

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
@ioannisg - this one and the one above do not seem exactly miscellaneous. Should they be included here or possibly in a separate PR?